docs/content/concepts/automation.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: "Automation | Dagster Docs"
+description: "Learn to automatically run your Dagster pipelines."
+---
+
+# Automation
+
+Dagster offers several ways to run data pipelines without manual intervation, including traditional scheduling and event-based triggers. Automating your Dagster pipelines can boost efficiency and ensure that data is produced consistently and reliably.
+
+When one of Dagster's automation methods is triggered, a **tick** is created, which indicates that a **run** should occur. The tick will kick off a run, which is a single instance of a pipeline being executed.
+
+In this guide, we'll cover the available automation methods Dagser provides and when to use each one.
+
+---
+
+## Prerequisites
+
+Before continuing, you should be familiar with:
+
+- [Software-defined Assets][assets]
+- [Jobs][jobs] (_optional_)
+- [Ops][ops] (_optional; advanced_)
+
+---
+
+## Available methods
+
+In this section, we'll touch on each of the automation methods currently supported by Dagster. After that we'll discuss what to think about when [selecting a method](#selecting-a-method). 
+
+### Schedules
+
+Schedules kick off [jobs][jobs] at fixed time intervals. Using schedules, you can start jobs at specific times, such as Mondays at 9:00 AM. Jobs triggered by schedules can contain a subset of [assets][assets] or [ops][ops]. Refer to the [Schedules documentation][schedules] to learn more.
+
+### Sensors
+
+To run a job or materialize an asset in response to specific events, you can use sensors. Sensors continuously check and execute logic to know whether to kick off a run. They are commonly used for situations where you want to materialize an asset after something else happens, such as:
+
+- A new file arrives in a specific location, such as Amazon S3
+- Another asset elsewhere has materialized
+- An external system frees up a worker slot
+
+You can also use sensors to act on the status of a job run. Refer to the [Sensors documentation][sensors] to learn more.
+
+### Auto-materialize policies <Experimental />
+
+Unlike schedules, Auto-materialize Policies (AMP) don't run at a specific time. Instead, an AMP allows you to describe the conditions under which assets should be materialized or skipped. An AMP, for example, could kick off a run for a downstream asset immediately after upstream changes occur.
```

Comment:
Technically, there are AMP that have cron-schedules. Not sure if that's a necessary point here though